doc/classes/Vector4i.xml

```diff
@@ -1,27 +1,33 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Vector4i" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Vector used for 4D math using integer coordinates.
 	</brief_description>
 	<description>
+		4-element structure that can be used to represent 4D grid coordinates or sets of integers.
+		It uses integer coordinates. See [Vector4] for its floating-point counterpart.
 	</description>
 	<tutorials>
 	</tutorials>
 	<constructors>
 		<constructor name="Vector4i">
 			<return type="Vector4i" />
 			<description>
+				Constructs a default-initialized [Vector4i] with all components set to [code]0[/code].
 			</description>
 		</constructor>
 		<constructor name="Vector4i">
 			<return type="Vector4i" />
 			<param index="0" name="from" type="Vector4i" />
 			<description>
+				Constructs a [Vector4i] as a copy of the given [Vector4i].
 			</description>
 		</constructor>
 		<constructor name="Vector4i">
 			<return type="Vector4i" />
 			<param index="0" name="from" type="Vector4" />
 			<description>
+				Constructs a new [Vector4i] from [Vector4].
```

Comment:
```suggestion
				Constructs a new [Vector4i] from the given [Vector4].
```

Reply:
Done

Reply:
Should be "the given", not just "given".

Reply:
Well, I guess today is one of those days… Better not touch anything else for the rest of the day

Did it right this time